File: URIS2023-AgilIS-Tim10-main/BacklogItem_MicroService/BacklogItem_MicroService/Models/Entities/StoryPoint.cs
using BacklogItem_MicroService.Models.Enums;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace BacklogItem_MicroService.Models.Entities
{
    public class StoryPoint
    {
        [Key]
        public Guid StoryPointId { get; set; }

        [Required]
        [EnumDataType(typeof(StoryPointEnum))]
        public StoryPointEnum StoryPointValue { get; set; }

        public Guid BacklogItemId { get; set; }
        public BacklogItem BacklogItem { get; set; }
    }
}

File: URIS2023-AgilIS-Tim10-main/URIS2023_TempateProject-main/src/app/models/story-point.ts
import { BacklogItem } from "./backog-item"

export class StoryPoint {
   storyPointId: string
   storyPointValue: number
   backlogItemId: string
   backlogItem: BacklogItem
}

export class StoryPointCreation {
    storyPointValue: number
     backlogItemId: string

   constructor(storyPointValue, backlogItemId ){
    this.storyPointValue=storyPointValue;
    this.backlogItemId =backlogItemId
   }
}

export class StoryPointUpdate {
    storyPointId: string
    storyPointValue: number

    constructor(storyPointId, storyPointValue ){
      this.storyPointId =storyPointId
      this.storyPointValue=storyPointValue;
       }

 }